resources/js/contact.js

```diff
@@ -1,171 +1,171 @@
 // https://youtu.be/eg4e-FObyJ8
 const submitBtn = document.querySelector('.formBtn')
-const fields = document.querySelectorAll('.input');
+const fields = document.querySelectorAll('.input') 
 const form = document.querySelector('form')
 
 var isValid = true
 
-submitBtn.addEventListener('click', function(){
+submitBtn.addEventListener('click', function () {
   validateFields(fields)
-  
+
   if (isValid) {
-  
+
     var contactData = {
       name: fields[0].value,
       email: fields[3].value,
       phone: fields[2].value,
       message: fields[4].value,
-    };
-    
-    const contactJSON = JSON.stringify(contactData);
+    } 
+
+    const contactJSON = JSON.stringify(contactData) 
     // clearError()
-    apiCall(contactJSON);
+    apiCall(contactJSON) 
   } else {
-    isValid = true;
+    isValid = true 
   }
-  
+
 })
 
-function validateFields(fields){
-  
-  for(i=0; i<fields.length;i++){
+function validateFields(fields) {
+
+  for (i = 0  i < fields.length  i++) {
```

Comment:
Missing semi-colons between for statements

Must be cause of your formatter extension in place

Reply:
Replaced for loops with for...of.